### DISCUSSION TITLE
Maui BadgeView

### DISCUSSION THREAD

**Discussion by Kremed** [CommunityToolkit/Maui#856] [Q&A] [answered]
I was wondering if the Maui Community Toolkit has Badge View now ?
also where can I look for a roadmap of features to be built !?

**Comment by bijington** [comment, marked as answer]
`BadgeView` is on the list to be done but I don't believe much progress has been made.

As for the roadmap you can find details in our readme https://github.com/CommunityToolkit/Maui#roadmap--plan

**Comment by VladislavAntonyuk** [comment]
I will start working on it after Dialogs PR is merged. 
for now you can add it to your project following this tutorial: https://vladislavantonyuk.github.io/articles/AppIcon-Badge-with-.NET-MAUI

**Comment by RobbiewOnline** [comment]
In Xamarin Forms there was BadgeView as part of xmlns:xct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit" ([see original docs)](https://learn.microsoft.com/en-us/xamarin/community-toolkit/views/badgeview), this enabled me to render _within_ my app a badge, but it's not in MAUI's community toolkit (yet?).  

The answer from @VladislavAntonyuk looks like an Android operating system badge for the home screen (_outside_ of the app).
 
I'm looking for a BadgeView alternative and stumbled on this discussion, is this the right place for what I'm looking for?

**Reply by bijington** [reply]
The proposal for BadgeView is here https://github.com/CommunityToolkit/Maui/issues/105 it looks like it was started by a member of the community but never finished.

**Reply by RobbiewOnline** [reply]
Thanks, that issue lead me to https://github.com/CommunityToolkit/Maui/pull/345 but that was abandoned.

Is there any similar view that you know of that exists within MAUI / Community Toolkit, or is under active development?

Thank you.

**Reply by bijington** [reply]
> Thanks, that issue lead me to https://github.com/CommunityToolkit/Maui/pull/345 but that was abandoned.
> 
> 
> 
> Is there any similar view that you know of that exists within MAUI / Community Toolkit, or is under active development?
> 
> 
> 
> Thank you.

Sadly not. I believe we need to revive whatever was in that PR or start afresh. Is this something you would be interested in taking on?

I don't know about others but my current plan is to tidy up any Popup related issues and then finish off the CameraView work.

**Reply by bijington** [reply]
> Thanks, that issue lead me to https://github.com/CommunityToolkit/Maui/pull/345 but that was abandoned.
> 
> 
> 
> Is there any similar view that you know of that exists within MAUI / Community Toolkit, or is under active development?
> 
> 
> 
> Thank you.

Sadly not. I believe we need to revive whatever was in that PR or start afresh. Is this something you would be interested in taking on?

I don't know about others but my current plan is to tidy up any Popup related issues and then finish off the CameraView work.

**Reply by bijington** [reply]
> Thanks, that issue lead me to https://github.com/CommunityToolkit/Maui/pull/345 but that was abandoned.
> 
> 
> 
> Is there any similar view that you know of that exists within MAUI / Community Toolkit, or is under active development?
> 
> 
> 
> Thank you.

Sadly not. I believe we need to revive whatever was in that PR or start afresh. Is this something you would be interested in taking on?

I don't know about others but my current plan is to tidy up any Popup related issues and then finish off the CameraView work.

**Reply by bijington** [reply]
> Thanks, that issue lead me to https://github.com/CommunityToolkit/Maui/pull/345 but that was abandoned.
> 
> 
> 
> Is there any similar view that you know of that exists within MAUI / Community Toolkit, or is under active development?
> 
> 
> 
> Thank you.

Sadly not. I believe we need to revive whatever was in that PR or start afresh. Is this something you would be interested in taking on?

I don't know about others but my current plan is to tidy up any Popup related issues and then finish off the CameraView work.

**Reply by bijington** [reply]
> Thanks, that issue lead me to https://github.com/CommunityToolkit/Maui/pull/345 but that was abandoned.
> 
> 
> 
> Is there any similar view that you know of that exists within MAUI / Community Toolkit, or is under active development?
> 
> 
> 
> Thank you.

Sadly not. I believe we need to revive whatever was in that PR or start afresh. Is this something you would be interested in taking on?

I don't know about others but my current plan is to tidy up any Popup related issues and then finish off the CameraView work.

**Reply by bijington** [reply]
Well that ended weirdly. Sorry the GitHub reported that the comment couldn't be saved so it looks like there are 5 copies now

**Comment by RobbiewOnline** [comment]
Unfortunately I don't have the bandwidth right now, I'm in the middle of converting three Xamarin.Forms projects to MAUI.  I don't have the experience of MAUI or XF under-the-hood to confidently say yes, but once I've migrated my projects then I will re-assess.

**Comment by CliffAgius** [comment]
Closed as answered
